plugin/db/pg/pg.go

```diff
@@ -443,6 +457,26 @@ func (driver *Driver) Execute(ctx context.Context, statement string) error {
 	return nil
 }
 
+func hasDatabaseInStatement(databases []*pgDatabaseSchema, createDatabaseStatement string) (bool, error) {
```

Comment:
It feels like it can be split into two functions, one for parsing database name and the other for checking if it's in the list (maybe not needed)

Reply:
Yeah, that's my initial thought but I decided to save a function and the combined function is simple enough with unit test coverage.

Reply:
The main thing is that the function name looks strange and is a bit different from what it actually does

Reply:
Done.